app/controllers/issues_controller.rb

```diff
@@ -121,6 +123,13 @@ def import
   end
 
   private
+  def all_evidence_columns
+    @issue.evidence
+          .map { |evidence| evidence.fields.keys - ['Title', 'Label']  }
```

Comment:
can you move the `- []` part of the operation out of the loop? That way instead of it being applied to each iteration we just apply it once, after `.uniq`.

Reply:
makes sense, pushed the change.